### PR TITLE
added error message for log backup

### PR DESCRIPTION
**Pull request by YuJuncen**
The basic for error handing in log backup.

Signed-off-by: Yu Juncen <yujuncen@pingcap.com>

### CODE REVIEW

File: proto/brpb.proto [hunk 1]
```diff
@@ -412,3 +412,12 @@ message DataFileInfo {
     // It may support encrypting at future.
     reserved "iv";
 }
+
+message LogBackupError {
```
Comment (3pointer) [inline proto/brpb.proto:416]: I preferred unified the name. LogBackup or StreamBackup

File: proto/brpb.proto [hunk 2]
```diff
@@ -412,3 +412,12 @@ message DataFileInfo {
...
+    // the unix epoch time (in millisecs) of the time the error reported.
+    uint64 happen_at = 1;
+    // the unified error code of the error.
+    string error_code = 2;
+    // the user-friendly error message.
+    string error_message = 3;
```
Comment (joccau) [inline proto/brpb.proto:422]: error_message contains store_id and error reason ?

Comment (YuJuncen) [reply proto/brpb.proto:422]: store id should already be in the key path, anyway we can add it here.

**Review by 3pointer — APPROVED** [review]

**Review by joccau — APPROVED** [review]
LGTM

**Pull request merged**